addon/components/pie-chart.js

```diff
@@ -366,9 +366,12 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         value = data.value;
       }
       formatLabelFunction = this.get('formatLabelFunction');
-      content = "<span class=\"tip-label\">" + data.label + "</span>";
-      content += "<span class=\"name\">" + (this.get('tooltipValueDisplayName')) + ": </span>";
-      content += "<span class=\"value\">" + (formatLabelFunction(value)) + "</span>";
+
+      var toolValueDisplayName = $("<span class=\"name\" />").text(this.get('tooltipValueDisplayName') + ": ");
+      var tipLabel = $("<span class=\"tip-label\"/>").text(data.label);
+      content = $("<span />").append(tipLabel)
+      .append(toolValueDisplayName)
+      .append("<span class=\"value\">" + (formatLabelFunction(value)) + "</span>");
       return this.showTooltip(content, d3.event);
```

Comment:
I spent some time playing around with this after you left my desk, and came up with the below.

It looks a little more readable IMO, and it also doesn't leave in the dummy `<SPAN>` element in the final HTML (thanks to the `content.html()` call at the end).

Analogous changes would also apply to the other code changes in `addon/`, of course.

```javascript
content = $('<span>');
content.append($('<span class="tip-label">').text(data.label));
content.append($('<span class="name">').text(this.get('tooltipValueDisplayName') + ': '));
content.append($('<span class="value">').text(formatLabelFunction(value)));
this.showTooltip(content.html(), d3.event);
```